dotnetv3/Glue/Actions/GlueWrapper.cs

```diff
@@ -0,0 +1,361 @@
+    {
+        var databasesRequest = new GetDatabaseRequest
+        {
+            Name = dbName,
+        };
+
+        var response = await _amazonGlue.GetDatabaseAsync(databasesRequest);
+        return response.Database;
+    }
+
+    // snippet-end:[Glue.dotnetv3.GetDatabase]
+
+    // snippet-start:[Glue.dotnetv3.GetJobRun]
+    /// <summary>
+    /// Get information about a specific AWS Glue job run.
+    /// </summary>
+    /// <param name="jobName">The name of the job.</param>
+    /// <param name="jobRunId">The Id of the job run.</param>
+    /// <returns>A JobRun object with information about the job run.</returns>
+    public async Task<JobRun> GetJobRunAsync(string jobName, string jobRunId)
+    {
+        var response = await _amazonGlue.GetJobRunAsync(new GetJobRunRequest { JobName = jobName, RunId = jobRunId });
+        return response.JobRun;
+    }
+
+    // snippet-end:[Glue.dotnetv3.GetJobRun]
+
+    // snippet-start:[Glue.dotnetv3.GetJobRuns]
+    /// <summary>
+    /// Get information about all AWS Glue runs of a specific job.
+    /// </summary>
+    /// <param name="jobName">The name of the job.</param>
+    /// <returns>A list of JobRun objects.</returns>
+    public async Task<List<JobRun>> GetJobRunsAsync(string jobName)
+    {
+        var jobRuns = new List<JobRun>();
+
+        var request = new GetJobRunsRequest
+        {
+            JobName = jobName,
+        };
+
+        do
+        {
+            var response = await _amazonGlue.GetJobRunsAsync(request);
+            jobRuns.AddRange(response.JobRuns);
+            if (response.NextToken is not null)
+            {
+                request.NextToken = response.NextToken;
+            }
+
+        } while (request.NextToken is not null);
+
+        return jobRuns;
+    }
+
+    // snippet-end:[Glue.dotnetv3.GetJobRuns]
+
+    // snippet-start:[Glue.dotnetv3.GetTables]
+    /// <summary>
+    /// Get a list of tables for an AWS Glue database.
+    /// </summary>
+    /// <param name="dbName">The name of the database.</param>
+    /// <returns>A list of Table objects.</returns>
+    public async Task<List<Table>> GetTablesAsync(string dbName)
+    {
+        var request = new GetTablesRequest { DatabaseName = dbName };
+        var tables = new List<Table>();
+
+        do
+        {
+            var response = await _amazonGlue.GetTablesAsync(request);
+            tables.AddRange(response.TableList);
+            if (response.NextToken is not null)
+            {
+                request.NextToken = response.NextToken;
+            }
+        } while (request.NextToken is not null);
+
+        return tables;
+    }
+
+    // snippet-end:[Glue.dotnetv3.GetTables]
+
+    //snippet-start:[Glue.dotnetv3.ListJobs]
+    /// <summary>
+    /// List AWS Glue jobs using a paginator.
+    /// </summary>
+    /// <returns>A list of AWS Glue job names.</returns>
+    public async Task<List<string>> ListJobsAsync()
```

Comment:
This method has no references or tests, should it be called in the scenario and the integration tests?

Reply:
It is part of the list of actions in the EPIC and it is listed in the metadata.

Reply:
But is it called anywhere? It needs to be called in the tests at the very least if it's included in the PR.

Reply:
Added a test for the method.